README.md

```diff
@@ -43,6 +43,23 @@ Then deploy the application:
 helm upgrade --install -n $NAMESPACE trustify charts/trustify --values values-minikube.yaml --set-string appDomain=$APP_DOMAIN
 ```
 
+> With traces enabled:
+
+
+```bash
+helm upgrade --install --dependency-update -n $NAMESPACE infrastructure charts/trustify-infrastructure --values values-minikube.yaml --set-string keycloak.ingress.hostname=sso$APP_DOMAIN --set-string appDomain=$APP_DOMAIN --set jaeger.enabled=true --set-string jaeger.allInOne.ingress.hosts[0]=jaeger$APP_DOMAIN --set tracing.enabled=true --timeout 10m
+```
+
+Get the OTEL collector service name:
+
+```bash
+kubectl get svc
```

Comment:
I guess there's something missing. This would fetch all services, right?

Reply:
updated this part thanks :+1: